lib/Element.js

```diff
@@ -18,6 +14,22 @@ function styleCamelCase (name) {
   }
 }
 
+function isTruthy (value) {
+  return !!value
+}
+
+function isString (value) {
+  return typeof value === 'string'
+}
+
+function isUndefined (value) {
+  return typeof value === 'undefined'
+}
+
+function clone (value) {
+  return Object.assign({}, value)
```

Comment:
Yep, you're right. I was just going for "remove dependency on lodash" first then look into browser support. This will be a major version bump too, just in case I have to drop any support for anything by accident. I'll check out the polyfills after I'm done replacing things.

I'm glad you liked it, I hope it comes in useful! I'm using it in production and I think others are too (99% sure). Let me know if you need help, or raise issues. If you do end up using it maybe I could start a list of "people who used it for things" in the readme too.